package.json

```diff
@@ -10,6 +10,7 @@
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "@usedapp/core": "^0.5.5",
```

Comment:
Yes yes, the march toward ethers begins!
![muahaha](https://media.giphy.com/media/eHYazg6wGDqYE/giphy.gif)